koji_wrapper/base.py

```diff
@@ -0,0 +1,53 @@
+# -*- coding: utf-8 -*-
+
+"""
+Provides common functionality for classes in this module which
+wrap a connection to koji, manage the session, and provide
+convenience methods for interacting with the koji api.
+"""
+
+import koji
+import operator
+
+class KojiWrapperBase(object):
+
+    def __init__(self, url='', topurl='', session=None):
```

Comment:
We should be using self.__<property name> here so that the below getters and setters behave as expected.

Reply:
I missed the session verification piece at the bottom. This pattern was mention in https://stackoverflow.com/questions/2825452/correct-approach-to-validate-attributes-of-an-instance-of-class#2825580

This is OK to merge.